web/src/components/modal-event-details/index.tsx

```diff
@@ -335,6 +330,9 @@ export const ModalEventDetails: React.FC<IModalEventDetails> = ({
         const selectedEventAddress = selectedEvent?.address;
         setValue('startTimeValue', selectedEvent.startTime);
         setValue('endTimeValue', selectedEvent.endTime);
+        // Reset the date explictly to today this will allow the useEffect
+        // to set the startTime and endTime values to the correct DateTime
+        setValue('dateValue', dayjs().toDate());
```

Comment:
I'm curious: what's wrong with `new Date()`?

Reply:
Nothing, both are equivalent

Reply:
Then please use new Date() to avoid confusion :)